js-packages/@prestojs/viewmodel/src/ViewModelFactory.ts

```diff
@@ -651,6 +657,11 @@ export default function viewModelFactory<T extends FieldsMapping>(
                 return this[pkFieldName];
             },
         },
+        _key: {
+            get(): PrimaryKey {
+                return this._pk;
```

Comment:
Is `_pk` still needed at all? Can we just replace it entirely with `_key`?

Reply:
Mainly because it was already there and used throughout + parallel to django's `record.pk`

i guess it would make sense to remove it and just use _key